src/o365/spo/commands/page/page-clientsidewebpart-add.ts

```diff
@@ -0,0 +1,483 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import * as request from 'request-promise-native';
+import commands from '../../commands';
+import { CommandOption, CommandValidate } from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+import GlobalOptions from '../../../../GlobalOptions';
+import { Auth } from '../../../../Auth';
+import {
+	ClientSidePage,
+	CanvasSection,
+	ClientSideWebpart,
+	ClientSidePageComponent,
+	CanvasColumn
+} from './clientsidepages';
+import { StandardWebPart, StandartWebPartUtils } from './StandardWebpartTypes';
```

Comment:
Could you rename the `StandardWebpartTypes` file to `StandardWebPartTypes` for consistent spelling? 🙂